src/utils/telemetry/report-error.mjs

```diff
@@ -0,0 +1,52 @@
+import { dirname, join } from 'path'
+import process, { version as nodejsVersion } from 'process'
+import { fileURLToPath } from 'url'
+
+import execa from '../execa.mjs'
+import getGlobalConfig from '../get-global-config.mjs'
+
+import { isTelemetryDisabled, cliVersion } from './utils.mjs'
+
+const dirPath = dirname(fileURLToPath(import.meta.url))
+
+/**
+ *
+ * @param {import('@bugsnag/js').NotifiableError} error
+ * @param {object} config
+ * @param {import('@bugsnag/js').Event['severity']} config.severity
+ * @returns {Promise<void>}
+ */
+export const reportError = async function (error, config = {}) {
+  const globalConfig = await getGlobalConfig()
+
+  if (isTelemetryDisabled(globalConfig)) {
+    return
+  }
+
+  const user = globalConfig.get(`users.${globalConfig.get('userId')}`)
+
+  const options = JSON.stringify({
+    type: 'error',
+    data: {
+      message: error.message,
+      name: error.name,
+      stack: error.stack,
+      cause: error.cause,
+      severity: config.severity,
+      user: {
+        id: user.id,
+        name: user.name,
+        email: user.email,
```

Comment:
IMHO I think this is not okay to track (name, email). Wouldn't userid be enough?

Reply:
talked with legal it would be ok to report it but I've dropped it as it's not super necessary.